debian/control

```diff
@@ -2,15 +2,13 @@ Source: felix
 Section: net
 Priority: optional
 Maintainer: Project Calico Maintainers <maintainers@projectcalico.org>
-Build-Depends: debhelper (>= 8.0.0), dh-python, dh-systemd, python-all (>= 2.7), python-setuptools
-XS-Python-Version: >= 2.7
+Build-Depends: debhelper (>= 8.0.0), dh-systemd
 Standards-Version: 3.9.4
 
 Package: calico-common
 Architecture: all
 Depends:
- ${misc:Depends},
- ${shlibs:Depends}
+ ${misc:Depends}
```

Comment:
I think it means "shared library deps"; I don't think we were using it (and debhelper was throwing a warning saying it didn't know what it was).